src/ring_theory/subsemiring.lean

```diff
@@ -369,6 +425,32 @@ lemma closure_eq_of_le {s : set R} {t : subsemiring R} (h₁ : s ⊆ t) (h₂ :
   closure s = t :=
 le_antisymm (closure_le.2 h₁) h₂
 
+lemma to_subsemiring_coe :
+  (M.to_subsemiring : set R) = add_submonoid.closure (M : set R) := rfl
```

Comment:
I think this lemma has the wrong name - it's currently called `subsemiring.to_subsemiring_coe`, but I assume it should be `submonoid.to_subsemiring_coe`?

Reply:
Yes, I think so